leaderboard.py

```diff
@@ -125,94 +120,94 @@ def update_filtered_leaderboard(tab, summary_lang: str, source_lang: str,
-      with gr.Accordion("Filter", open=False) as summarization_filter:
-        with gr.Row():
-          languages = [
-              language.name.capitalize() for language in lingua.Language.all()
-          ]
-          summary_language = gr.Dropdown(
-              choices=languages,
-              value=DEFAULT_FILTER_OPTIONS["summary_language"],
-              label="Summary language",
-              interactive=True)
-
-        with gr.Row():
-          filtered_summarization = gr.DataFrame(
-              headers=["Rank", "Model", "Elo rating"],
-              datatype=["number", "str", "number"],
-              value=lambda: load_elo_ratings(
-                  LeaderboardTab.SUMMARIZATION, DEFAULT_FILTER_OPTIONS[
-                      "summary_language"]),
-              elem_classes="leaderboard")
-
-      summary_language.change(fn=update_filtered_leaderboard,
-                              inputs=[
-                                  gr.State(LeaderboardTab.SUMMARIZATION),
-                                  summary_language,
-                                  gr.State(),
-                                  gr.State()
-                              ],
-                              outputs=filtered_summarization)
-
-      gr.Dataframe(headers=["Rank", "Model", "Elo rating"],
-                   datatype=["number", "str", "number"],
-                   value=lambda: load_elo_ratings(LeaderboardTab.SUMMARIZATION),
-                   every=LEADERBOARD_UPDATE_INTERVAL,
-                   elem_classes="leaderboard")
+      summary_language = gr.Dropdown(choices=SUPPORTED_LANGUAGES +
+                                     [ANY_LANGUAGE],
+                                     value=ANY_LANGUAGE,
+                                     label="Summary language",
+                                     interactive=True)
+
+      filtered_summarization = gr.DataFrame(
+          headers=["Rank", "Model", "Elo rating"],
+          datatype=["number", "str", "number"],
+          value=lambda: load_elo_ratings(LeaderboardTab.SUMMARIZATION,
+                                         ANY_LANGUAGE),
+          elem_classes="leaderboard",
+          visible=False)
+
+      original_summarization = gr.Dataframe(
+          headers=["Rank", "Model", "Elo rating"],
+          datatype=["number", "str", "number"],
+          value=lambda: load_elo_ratings(LeaderboardTab.SUMMARIZATION),
+          every=LEADERBOARD_UPDATE_INTERVAL,
+          elem_classes="leaderboard")
       gr.Markdown(LEADERBOARD_INFO)
 
+      summary_language.change(
+          fn=update_filtered_leaderboard,
+          inputs=[
+              gr.State(LeaderboardTab.SUMMARIZATION), summary_language,
+              gr.State(),
+              gr.State()
+          ],
+          outputs=filtered_summarization).then(
+              fn=toggle_leaderboard,
+              inputs=summary_language,
+              outputs=[original_summarization, filtered_summarization])
```

Comment:
I checked `toggle_leaderboard` and it seems it create a new Dataframe with only `visible` field?
Does `gradio` automatically merge the value with `original_summarization` and `filtered_summarization`?

Reply:
`toggle_leaderboard` only updates visibility of existing original_summarization dataframe and filtered_summarization dataframe